controller/api/tests/test_config.py

```diff
@@ -509,3 +551,125 @@ def test_unauthorized_user_cannot_modify_config(self):
         response = self.client.post(url, json.dumps(body), content_type='application/json',
                                     HTTP_AUTHORIZATION='token {}'.format(unauthorized_token))
         self.assertEqual(response.status_code, 403)
+
+    def _test_app_healthcheck(self,):
```

Comment:
surprisingly flake8 doesn't care about this, but it doesn't break anything. I'll follow up on this one.